cli/foundation-models/nvidia-nemo/nemo_pipeline_job.yml

```diff
@@ -0,0 +1,60 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: correct_test_task_type
+description: correct task type testing
+
+
+inputs:
+  compute: do-not-delete-nvidia-testing
+  model_path: 
+    type: triton_model
+    path: azureml://registries/nvidia-ai/models/Nemotron-3-8B-Base-4k/versions/1 
+  train_dataset_path: 
+    type: uri_folder
+    path: azureml:train_tg_nv:1
```

Comment:
@vikasagrawal-ms  Do we have this as part of the notebook?

Reply:
in notebook we are using samsum dataset